src/entities/Attendance.ts

```diff
@@ -25,7 +25,7 @@ export class Attendance {
   @Column('timestamp', { nullable: true })
   endTime?: Date;
 
-  @Column({ nullable: true })
+  @Column({ type: 'float4', nullable: true })
```

Comment:
This gives 6 decimal degits precision. Any reason why you picked this? Just curious.

Reply:
it works :)